File: BA.Grisecorp.App.NetCore21/BA.Grisecorp.App.Infra.Data/Repository/GrupoEtapaRepository.cs
using BA.Grisecorp.App.API.Domain.Aggregates.GrupoEtapa.Grupo;
using BA.Grisecorp.App.API.Domain.Aggregates.GrupoEtapa.Grupo.Repository;
using BA.Grisecorp.App.Infra.Data.Context;
using BA.Grisecorp.App.Infra.Data.Core.Repository;
using System;
using System.Collections.Generic;
using System.Text;

namespace BA.Grisecorp.App.Infra.Data.Repository
{
    public class GrupoEtapaRepository : Repository<GrupoEtapa>, IGrupoEtapaRepository
    {
        public GrupoEtapaRepository(DatabaseContext context) : base(context)
        {

        }
    }
}

File: webapi/src/webapi/Data/Repository/GrupoEtapaRepository.php
<?php
namespace webapi\Data\Repository;

use webapi\Data\Core\Repository\Repository;
use webapi\Domain\Aggregates\GrupoEtapa\GrupoEtapa\Repository\IGrupoEtapaRepository;
class GrupoEtapaRepository  extends Repository implements IGrupoEtapaRepository
{

	public function __construct() {
		parent::__construct('webapi\Domain\Aggregates\GrupoEtapa\GrupoEtapa\GrupoEtapa');
	}
}


?>